Answers/Drawing Solomon Temple.py

```diff
@@ -0,0 +1,144 @@
+    #B=turtle.Turtle()
+    #C=turtle.Turtle()
+    porchTurtle.hideturtle() #hides the cursor immediately.
+    return wn
+
+
+def drawPorch(turtle): # I need a turtle object as a parameter here, using 't'.
+    turtle.begin_fill()
+    turtle.fillcolor("red")
+    turtle.forward(250) #Line 22 refs. line 21.
+    turtle.right(90)
+    turtle.forward(100)
+    turtle.right(90)
+    turtle.forward(350)
+    turtle.right(90)
+    turtle.forward(100)
+    turtle.right(90)
+    turtle.forward(100)
+    turtle.end_fill()
+    """DOCSTRING PLACEHOLDER"""
+#    porchTurtle = turtle.Turtle() #Is the object properly defined?
+#    turtle.forward(100)
+#    return porchTurtle
+
+
+def drawPillarOne(turtle):
+    """DOCSTRING PLACEHOLDER"""
+    turtle.fillcolor = "white"
+    turtle.begin_fill()
+    turtle.left(90)
+    turtle.forward(100)
+    turtle.left(90)
+    turtle.forward(50)
+    turtle.left(90)
+    turtle.forward(100)
+    turtle.end_fill()
+    #turtle.forward(100)
+    #turtle.left(90)
+    #turtle.forward(50)
+    #turtle.left(90)
+    #turtle.forward(100)
+    #turtle.left(90)
+    #turtle.forward(70)
+    #turtle.right(90)
+    #turtle.forward(10)
+    ##turtle.right(90)
+    #turtle.forward(80)
+    #turtle.right(90)
+    #turtle.forward(10)
+    #turtle.right(90)
+    #turtle.forward(80)
+
+
+
+def drawPillarTwo(turtle):
+    """DOCSTRING PLACEHOLDER"""
+    turtle.fillcolor = "black"
+    turtle.begin_fill()
+    turtle.left(90)
+    turtle.forward(100)
+    turtle.left(90)
+    turtle.forward(50)
+    turtle.left(90)
+    turtle.forward(100)
+    turtle.end_fill()
+    #turtle.fillcolor = "black"
+    #turtle.begin_fill()
+    #turtle.left(90)
+    #turtle.forward(100)
+    #turtle.left(90)
+    #turtle.forward(50)
+    #turtle.left(90)
+    #turtle.forward(100)
+    #turtle.left(90)
+    #turtle.forward(50)
+    #turtle.left(90)
+    #turtle.forward(100)
+    #turtle.left(90)
+    #turtle.forward(70)
+    #turtle.right(90)
+    #turtle.forward(10)
+    #turtle.right(90)
+    #turtle.forward(80)
+    #turtle.right(90)
+    #turtle.forward(10)
+    #turtle.right(90)
+    #turtle.forward(80)
+
+
+#def drawStar(turtle):
+    """DOCSTRING PLACEHOLDER"""
+#    turtle.forward(50)
+
+#def drawWindow(turtle):
+    """DOCSTRING PLACEHOLDER"""
+ #   turtle.forward(100)
+
+#def colorShape():
+    """DOCSTRING PLACEHOLDER"""
+
+def main():
+    """DOCSTRING PLACEHOLDER"""
+    wn = makeGUI()
+    pillarTurtleOne = turtle.Turtle()
+    pillarTurtleTwo = turtle.Turtle()
+    pillarTurtleOne.hideturtle()
+    #pillarTurtleOne.pensize(10)
+    pillarTurtleTwo.color("white")
+    pillarTurtleTwo.goto(200, 0)
+    pillarTurtleTwo.hideturtle()
+    porchTurtle = turtle.Turtle()
+    porchTurtle.hideturtle()
+    #starTurtle = turtle.Turtle()
+    #windowTurtle = turtle.Turtle()
+    # makeGUI() #This function isn't needed? The program still runs when I comment it out?
+    #turtle.Turtle() #Generates a turtle object using the .Turtle() method.
```

Comment:
Clean up your code (i.e., remove any commented out code, extra code that's not called/used, etc.)